frontend/public/components/storage/storage.js

```diff
@@ -0,0 +1,63 @@
+import * as _ from "lodash-es";
+import * as k8sModels from "../../models";
+
+export const createVolume = function(name, persistentVolumeClaim) {
+  return {
+    name: name,
+    persistentVolumeClaim: {
+      claimName: persistentVolumeClaim
+    }
+  };
+};
```

Comment:
This is so simple I'd probably just inline it, but prefer ES6 syntax

```js
export const createVolume (name, claimName) => ({
  name,
  persistentVolumeClaim: {
    claimName,
  },
});
```

I have similar comments for the JS below. Use fat arrow, const, ES6 property shorthand, etc. below. (I realize most of this code was taken from the previous console, but we should update it for ES6.)